spec/controllers/equipment_models_controller_spec.rb

```diff
@@ -1,421 +1,387 @@
-                             start_date: Time.zone.today - 1.day,
-                             due_date: Time.zone.today - 1.day,
-                             equipment_item: @model.equipment_items.first)
-          # needed to persist overdue_count
-          @model.save
-        end
-
-        it_behaves_like 'calculates availability correctly', 0
+          it_behaves_like 'calculates availability correctly', 0
+        end
       end
     end
-  end
 
-  describe 'GET new' do
-    context 'with admin user' do
-      before do
-        sign_in FactoryGirl.create(:admin)
-        get :new
-      end
-      it { is_expected.to respond_with(:success) }
-      it { is_expected.to render_template(:new) }
-      it { is_expected.not_to set_flash }
-      it 'assigns a new equipment model to @equipment_model' do
-        expect(assigns(:equipment_model)).to be_new_record
-        expect(assigns(:equipment_model)).to be_kind_of(EquipmentModel)
-      end
-      it 'sets equipment_model to nil when no category is specified' do
-        expect(assigns(:equipment_model).category).to be_nil
-      end
-      it 'sets category when one is passed through params' do
-        cat = model.category
-        get :new, category_id: cat
-        expect(assigns(:equipment_model).category).to eq(cat)
+    describe 'GET new' do
+      context 'with admin user' do
+        before do
+          mock_user_sign_in(mock_user(:admin))
+          get :new
+        end
+        it_behaves_like 'successful request', :new
+        it 'assigns a new equipment model to @equipment_model' do
+          expect(assigns(:equipment_model)).to be_new_record
+          expect(assigns(:equipment_model)).to be_kind_of(EquipmentModel)
+        end
+        it 'sets equipment_model to nil when no category is specified' do
+          expect(assigns(:equipment_model).category).to be_nil
+        end
+        it 'sets category when one is passed through params' do
+          get :new, category_id: model.category
+          expect(assigns(:equipment_model).category).to eq(model.category)
+        end
       end
-    end
-    context 'with non-admin user' do
-      before { sign_in FactoryGirl.create(:user) }
-      it 'should redirect to root' do
-        get :new
-        expect(response).to redirect_to(root_url)
+      context 'when not admin' do
+        before do
+          mock_user_sign_in
+          get :new
+        end
+        it_behaves_like 'redirected request'
       end
     end
-  end
 
-  describe 'GET edit' do
-    context 'with admin user' do
-      before do
-        sign_in FactoryGirl.create(:admin)
-        get :edit, id: model
-      end
-      it { is_expected.to respond_with(:success) }
-      it { is_expected.to render_template(:edit) }
-      it { is_expected.not_to set_flash }
-      it 'sets @equipment_model to selected model' do
-        expect(assigns(:equipment_model)).to eq(model)
+    describe 'GET edit' do
```

Comment:
considering that `EquipmentModelsController#edit` is empty, I think this can be deleted